packages/api/server.js

```diff
@@ -37,21 +33,9 @@ global.headCollection = process.env.HEAD_COLLECTION || 'testing-buildsget';
 app.use(cors());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true })); // support encoded bodies
-app.use(
-  session({
-    store: new FirestoreStore({
-      dataset: client,
-      kind: 'express-sessions'
-    }),
-    secret: process.env.SESSION_SECRET || 'no secret',
-    resave: false,
-    saveUninitialized: true
-  })
-);
 
 app.get('/api/repos', async (req, res) => {
-  const repository = new Repository(null);
-  const result = await repository.getCollection('dummy-repositories');
```

Comment:
I don't think we use this `/api/repos` route for anything at this point, and could probably just delete the whole thing?

@StuartRucker could you  confirm for me

Reply:
Yep this was a demo endpoint back when we were getting started. Stuart can confirm, but I'm fairly sure we don't use it for anything.

Reply:
Yeah it was replaced with /org/:org